build.sbt

```diff
@@ -1,27 +1,46 @@
 val scala2_12 = "2.12.15"
-val scala2_13 = "2.13.8"
+val scala2_13 = "2.13.12"
 
-val silencerVersion = "1.7.8"
+ThisBuild / scalaVersion       := scala2_13
+ThisBuild / majorVersion       := 9
+ThisBuild / isPublicArtefact   := true
 
 lazy val domain = (project in file("."))
+  .settings(publish / skip := true)
+  .aggregate(
+    play28,
+    play29,
+    play30
+  )
+
+lazy val commonSettings =
+  ScalariformSettings() ++ ScalariformSettings()
```

Comment:
I think this should be
`lazy val commonSettings = ScalariformSettings() ++ ScoverageSettings()`

Reply:
oh yeah, whoops - corrected in [this commit](https://github.com/hmrc/domain/pull/52/commits/84859f1ee5ab99b978f18e7c1e5f7e0e2e6c92b2)